docs/README.md

```diff
@@ -54,6 +54,34 @@ By default implementation we mean that whenever you use `checkout-summary` in yo
 
 Therefore, in order to customize the Checkout Summary configuration, you can simply use the default implementation in your code and change it as you wish.
 
```

Comment:
It would be great to clarify here which are the blocks exported by the `checkout-summary` app and which are the props they use (add a prop table). Since we are talking about an advanced configuration, users should be able to configure the `checkout-summary` block using this code stated above. But, effectively they can't once we are not clarifying which blocks/props are allowed.